src/main/java/com/alysaa/geyserupdater/common/util/GeyserProperties.java

```diff
@@ -42,8 +44,8 @@ public static String queryGitProperties(String propertyKey) throws IOException {
      * @return the latest build number
      * @throws UnsupportedEncodingException if failed to encode the given gitBranch
      */
-    public static int jenkinsLatestBuildNumber(String gitBranch) throws UnsupportedEncodingException {
-        String XML = WebUtils.getBody("https://ci.opencollab.dev/job/GeyserMC/job/Geyser/job/" + URLEncoder.encode(gitBranch, StandardCharsets.UTF_8.toString()) + "/lastSuccessfulBuild/api/xml?xpath=//buildNumber");
-        return Integer.parseInt(XML.replaceAll("<(\\\\)?(/)?buildNumber>", "").trim());
+    public static int getLatestGeyserBuildNumberFromJenkinsForGitBranch(String gitBranch) throws UnsupportedEncodingException {
```

Comment:
similarly here, `getLatestGeyserBuildNumberFromJenkins`

Reply:
Ah… yeah, that's just my own personal style when it comes to writing method/function names l o l

`doSomeTaskForArgumentName(WhateverType argumentName)`

Addressed by conforming to your stylistic choice in commit `5b4e9fa7b4aad55af9514accec6a2146d15c96e0`.